### PR TITLE
Tooltip themes

### PR DESCRIPTION
**Pull request by norbertprocak**

### CODE REVIEW

File: src/components/TooltipNew/style.scss [hunk 1]
```diff
@@ -4,7 +4,7 @@
   box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.3);
   font-size: 14px;
   border-radius: 4px;
-  padding: 12px;
+  padding: 12px 16px;
```
Comment (norbertprocak) [inline src/components/TooltipNew/style.scss]: This change was requested by Justyna :)

**Comment by kamilmateusiak** [comment]
Arrow of tooltips with the bottom placement is not visible at all: https://take.ms/07Ilt

File: src/components/TooltipNew/PopperTooltip.js [hunk 2]
```diff
@@ -340,6 +342,10 @@ class PopperTooltip extends React.PureComponent {
 PopperTooltip.propTypes = {
   children: PropTypes.node.isRequired,
   className: PropTypes.string,
+  /**
+   * The theme changes the look of the tooltip.
+   */
+  theme: PropTypes.oneOf(['normal', 'invert', 'important']),
```
Comment (kamilmateusiak) [inline src/components/TooltipNew/PopperTooltip.js]: IMHO we don't need `normal` theme. `popper-tooltip--normal` is redundant

File: src/components/TooltipNew/style.scss [hunk 3]
```diff
@@ -127,6 +139,18 @@
     visibility: visible;
   }
 
+  &--invert {
```
Comment (kamilmateusiak) [inline src/components/TooltipNew/style.scss]: You added styles for CssTooltip, but it's not used in the component. Check out `CssTooltip.js`

File: src/components/TooltipNew/design-guide.md [hunk 4]
```diff
@@ -18,3 +18,8 @@
 
   <img style="max-width: 500px; width: 100%;" src="./interactive_tooltip.jpg" alt="Interactive tooltip example" />
 </ul>
+
+<h3>Themes</h3>
+<p>You can choose one of the themes to modify the look of the Tooltip.</p>
```
Comment (kamilmateusiak) [inline src/components/TooltipNew/design-guide.md]: Can you add a `Select` with themes in the docs examples of tooltips?

File: src/interfaces/tooltipNew.d.ts [hunk 5]
```diff
@@ -3,11 +3,14 @@ import * as PopperJS from "popper.js";
...
 type RefHandler = (ref: HTMLElement | null) => void;
 
+export type TooltipTheme = "normal" | "invert" | "important";
+
 export type TooltipTriggerAction = "managed" | "click" | "hover";
 
 export interface IPopperTooltipProps
   extends React.HTMLAttributes<HTMLDivElement> {
   children: React.ReactNode;
+  theme?: TooltipTheme;
```
Comment (kamilmateusiak) [inline src/interfaces/tooltipNew.d.ts]: add the theme property to `ICssTooltipProps`

File: src/components/TooltipNew/PopperTooltip.js (hunk 2 above)
Comment (norbertprocak) [reply src/components/TooltipNew/PopperTooltip.js]: ok

File: src/components/TooltipNew/style.scss (hunk 3 above)
Comment (norbertprocak) [reply src/components/TooltipNew/style.scss]: ok

File: src/components/TooltipNew/design-guide.md (hunk 4 above)
Comment (norbertprocak) [reply src/components/TooltipNew/design-guide.md]: sure

File: src/interfaces/tooltipNew.d.ts (hunk 5 above)
Comment (norbertprocak) [reply src/interfaces/tooltipNew.d.ts]: ok

**Comment by norbertprocak** [comment]
> Arrow of tooltips with the bottom placement is not visible at all: https://take.ms/07Ilt

It's hard to do something about it. I would rather assume that the white tooltip is not intended to be shown on light backgrounds. We could discuss this with Justyna later.

File: src/interfaces/tooltipNew.d.ts (hunk 5 above)
Comment (kamilmateusiak) [reply src/interfaces/tooltipNew.d.ts]: still no `theme` in `ICssTooltipProps`

**Comment by kamilmateusiak** [comment]
Justyna orders us to change box-shadow of tooltips. Please use `0 4px 15px 0 rgba(0, 0, 0, 0.2)` :)

**Comment by norbertprocak** [comment]
> Justyna orders us to change box-shadow of tooltips. Please use `0 4px 15px 0 rgba(0, 0, 0, 0.2)` :)

ok

File: src/interfaces/tooltipNew.d.ts (hunk 5 above)
Comment (norbertprocak) [reply src/interfaces/tooltipNew.d.ts]: line 53

**Pull request merged**